alacritty/src/display/content.rs

```diff
@@ -366,10 +355,10 @@ impl RenderableCell {
     /// is computed.
     #[inline]
     fn compute_bg_alpha(config: &UiConfig, bg: Color) -> f32 {
-        if bg == Color::Named(NamedColor::Background) {
-            0.
-        } else if config.colors.transparent_background_colors {
+        if config.colors.transparent_background_colors {
             config.window_opacity()
```

Comment:
Is there a need to having this order for the checks? I wasn't able to understand why we do it this way

Reply:
If it's the same as the background color, we're not drawing anything anyway. So it doesn't matter what you set your `window_opacity` to. Making it *less* transparent than 0 will just cause it to be rendered, which is a waste of time since you can't see it anyway.